assets/js/blocks/cart-checkout/cart/full-cart/cart-line-items-table.js

```diff
@@ -10,9 +10,19 @@ import PropTypes from 'prop-types';
 import CartLineItemRow from './cart-line-item-row';
 
 const CartLineItemsTable = ( { lineItems = [] } ) => {
-	const products = lineItems.map( ( lineItem ) => (
-		<CartLineItemRow key={ lineItem.key } lineItem={ lineItem } />
-	) );
+	const products = lineItems.map( ( lineItemRaw ) => {
+		// convert low stock prop into camelCase
+		const {
+			low_stock_remaining: lowStockRemaining,
+			...lineItem
+		} = lineItemRaw;
+		return (
+			<CartLineItemRow
+				key={ lineItem.key }
+				lineItem={ { lowStockRemaining, ...lineItem } }
```

Comment:
A problem with this logic is that you're creating a new object every time this is rendered. Since it's likely the `lineItem` objects will not change often once retrieved, I don't think you should worry about this alias from `low_stock_remaining` to `lowStockRemaining`.  It's unnecessary logic and increases renders.  If you _want_ to do the aliasing, just do it in the destructuring of the `lineItem` object in the `CartLineItemRow` component.

Reply:
Gah! Hit the merge button on the wrong PR, without attending to this. I can revert or whip up a follow up PR - any preference @nerrad ? Sorry for the confusion.

Reply:
followup PR.  I do think it needs addressed because of the potential for unnecessary re-renders.

Reply:
or did you already revert?

Reply:
I'll do a follow up. Agree that this needs to be fixed up :)

I started a revert but will kill it, I think a revert would be more confusing and double handling.

Reply:
These fixes will be in #1564